src/applications/vaos/components/PastAppointmentsDateDropdown.jsx

```diff
@@ -34,7 +35,10 @@ export default function PastAppointmentsDateDropdown({
         className="vads-u-display--inline-block vads-u-margin-left--2"
         aria-label="Update my appointments list"
         disabled={currentRange === dateRangeIndex}
-        onClick={() => onChange(dateRangeIndex)}
+        onClick={() => {
+          onChange(dateRangeIndex);
+          focusElement(`#pastAppts`);
```

Comment:
@jbalboni Great question. Let's set focus after the data is finished loading, so users don't hear the announcement, and move off the spot without something to review.

Reply:
@jenniemc Trevor's answer means that you'll have to set the focus at a different point than right here. 

I believe the way to do this is instead of setting focus in this component, you'll need to do it in PastAppointmentsList, in a `componentDidUpdate` lifecycle method. You can compare the previous and current value of `appointments.pastStatus` and when it changes from loading to succeeded, set the focus.

https://reactjs.org/docs/react-component.html#componentdidupdate

Reply:
We do something similar here: https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/vaos/containers/AppointmentsPage.jsx#L38

Reply:
Hi @jbalboni, I made the changes to the componentDidUpdate but the focus doesn't seem to be working. I pushed the new changes out. Would you have a look at the PR?

Reply:
Hi @jbalboni, pushed the latest update to the PR. From Jenkins, it passed the entire pipline but the code coverage say 33% threshold. Could the sparse unit testing be trigger by other apps outside of VAOS?

Reply:
The 33% is calculated just against the changes in your PR, so I don't think so. It looks like the `componentDidUpdate` method you added isn't being covered in unit tests. You'll probably have to jump through some hoops to properly test it, since it'll require that you use `mount` instead of `shallow` and `tree.setProps` to cause an update that triggers your logic. I can try to find an example of a test like that tomorrow.

Reply:
@jenniemc Something like this: https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/vaos/tests/containers/ClinicChoicePage.unit.spec.jsx#L69

Reply:
@jbalboni, after making the changes to the unit test, based from the ClinicChoicePage example, I expected the active element node to equal 'h3' but ends up to be 'body'. When you get an opportunity, would you take a look at the unit test that I just push to the PR?